extensions/smallrye-health/deployment/src/main/java/io/quarkus/smallrye/health/deployment/SmallRyeHealthProcessor.java

```diff
@@ -52,10 +60,22 @@
     @ConfigRoot(name = "smallrye-health")
     static final class SmallRyeHealthConfig {
         /**
-         * The path of the health-checking servlet.
+         * The path of the legacy health-checking servlet.
```

Comment:
This is not entirely true. It's the root path of the probes.

I don't think making it the root path AND the legacy path will work as, as soon as you define the root path, you end up enabling the legacy probe whereas you might not want it.

I think I would get rid of the legacy probe entirely. We are still in early stages of Quarkus, I think we can do this sort of changes.

Reply:
If we do that Quarus won't support MP Health 2.0 specification since implementations must provide probe for `/health`. Do we really want that?

Reply:
Ah, OK. I wasn't aware of that. So, in this case, is it really legacy?

Reply:
It was kept in 2.0 for backward compatibility, so yes it’s a kind of legacy